applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -44,6 +44,9 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         elif (solver_type == "Embedded"):
             solver_module_name = "trilinos_navier_stokes_embedded_solver"
 
+        elif (solver_type == "TwoFluids"):
```

Comment:
Yes, I agree in this aspect - I often forget or add the "s" by mistake. The reason why I chose this keyword was because it coincides with the one used for the OpenMP version of the solver. I assume it would make sense to use the same keyword in both cases, wouldn't it? 
The serial solver class is called `class NavierStokesTwoFluidsSolver(FluidSolver)` by the way, where the "s" is present. Also the serial python file itself is named `navier_stokes_two_fluidS_solver.py`. Maybe it is not so bad to maintain the "s" and to alter the name of the Trilinos file and class name, instead.
@jcotela Would that be an option for you?
@rubenzorrilla May I also ask for your opinion?

Reply:
I would be OK with that, if it means that we have a consistent naming scheme (C, python, json, serial and trilinos).

Reply:
Alright. Now everything is named "...TwoFluids..." with "s"

Reply:
ok